config/locales/en.yml

```
@@ -655,6 +655,9 @@ en:
     python: Python
     rest: REST
 
+  oidc:
+    authorization_flow_warning: "Any changes to the authorization flow will only be applied to new applications; existing applications won't be updated."
```

I'd rephrase adding `default values` => that would work when we will do auth flow per application

```suggestion
    authorization_flow_warning: "New application will take default values from this authorization flow ; changing it will not update existing applications."
```

---

I don't understand the word 'default' in this context.

---

ping @hallelujah

---

we can change the phrase for CR1…lets just merge

---

```suggestion
    authorization_flow_warning: "Any changes to the authorization flow will only be applied to new applications; existing applications won't be affected."
```

---

Thanks. Based on Hery's feedback I'm now considering:

> Any changes to the authorization flow of this service will only be applied to new applications; existing applications will continue to use the original flow.